src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -99,22 +107,24 @@ Each issue preference option has its own advantages. You can select any preferen
 
       <td>
         An issue will be created for every incident in your policy.
```

Comment:
There is no setting that does this.

Reply:
I've replaced it with this: This option groups the incidents sharing the same condition and signal into their own issue.